src/lib/location/states.ts

```diff
@@ -9,8 +9,8 @@ export const states: Record<string, string> = {
 	'Connecticut': 'CT',
 	'Delaware': 'DE',
 	'Washington, D.C.': 'DC',
-	'District Of Columbia': 'DC',
-	'Federated States Of Micronesia': 'FM',
+	'District of Columbia': 'DC',
+	'Federated States of Micronesia': 'FM',
```

Comment:
I wonder why this one is even here, it's a country with its own ISO code, not a US state: https://ipinfo.io/119.252.112.0

Reply:
Removed